packages/docs/src/pages/components/upload/examples/ExSimple.vue

```diff
@@ -12,12 +12,14 @@
     </b-field>
 </template>
 
-<script>
-    export default {
```

Comment:
I do not think it is a good idea to fix the indentation in this PR, because it makes the diff unclear.